server_development/topics/providers.adoc

```diff
@@ -82,8 +82,6 @@ org.acme.provider.MyThemeSelectorProviderFactory
 ----
 
 You can configure your provider through `standalone.xml`, `standalone-ha.xml`, or `domain.xml`.  
-See the link:{installguide_link}[{installguide_name}] for more details on
```

Comment:
Do we really need to remove link to the installation guide? IMO keep the link can be useful, so that people know where to find the configuration files? If there is really no better way of fixing broken links, I agree to move it, but my vote would be to rather keep the link.

Reply:
It's not ideal. This is a compromise between different members of the team.  I think the interested readers will find it.
However, we have agreed to work on a long-term solution that works for CD releases and 7.x releases.

Reply:
Thanks, got it now. I agree with this change then.